cmd/bake/bake.go

```diff
@@ -19,11 +21,14 @@ type Options struct {
 	// Endpoint is the service endpoint to reach out to
 	Endpoint string
 
-	// URL is the git repo URL that will be sourced via 'pizza bake'
-	URL string
+	// URLs are the git repo URLs that will be source via 'pizza bake'
+	URLs []string
```

Comment:
The `bakeLongDesc` usage string also needs to be updated to reference taking multiple urls (not just one)